<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderForm</class>
 <widget class="QWidget" name="OrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0,1">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,0,1">
     <property name="spacing">
      <number>15</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLineEdit" name="edtTitle"/>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cbAgency"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cbAccountRep"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Client:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Agency:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edtClient">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Brand:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cbBrand"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Package:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Account Rep:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="cbPackage"/>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1" columnminimumwidth="0,0">
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="1" column="1">
        <widget class="QDateEdit" name="dtOrderDate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="sbSOrdered">
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>End Date:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Order Date:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="sbSPlayed">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sbOrderNumber">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="maximum">
          <number>99999999</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="sbSBooked">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateEdit" name="dtStartDate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Spots Played:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Spots Ordered:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Start Date:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Order Number:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDateEdit" name="dtEndDate">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Spots Booked:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Billing:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1">
          <item row="0" column="1">
           <widget class="QComboBox" name="cbRevenueType"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Billing Basis:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cbBillingPeriod"/>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="cbBillingBasis"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Revenue Type:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Billing Type:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cbBillingType"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Billing Period:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Commission:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="sbTradeCredit"/>
            </item>
            <item row="2" column="2">
             <widget class="QComboBox" name="cbSaleRepCommType"/>
            </item>
            <item row="1" column="2">
             <widget class="QComboBox" name="cbAgencyCommType"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Trade Credit:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Agency Comm:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="sbAgencyComm"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Sales Rep Comm:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QComboBox" name="cbTradeCreditType"/>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="sbSaleRepComm"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Discount:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="sbDiscount"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>% of Gross</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Charges:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_6" columnminimumwidth="0,1">
            <item row="0" column="0">
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Grace Period:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Late Fee:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="sbGracePeriod"/>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="sbLateFee"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
